ansible/roles/sidekiq/templates/sidekiq-monit-rc

```diff
@@ -0,0 +1,7 @@
+{% for queue in job_queues %}
+check process sidekiq_{{queue}}
+  with pidfile /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid
+  start program = "/bin/su - deploy -c 'cd /u/apps/{{project_name}}/current && bundle exec sidekiq --queue {{queue}} --pidfile /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid --environment {{rails_env}} --logfile /u/apps/{{project_name}}/shared/log/sidekiq.{{queue}}.log --daemon'" with timeout 30 seconds
+  stop program = "/bin/su - deploy -c 'kill -s TERM `cat /u/apps/{{project_name}}/shared/tmp/pids/sidekiq.{{queue}}.pid`'" with timeout 30 seconds
```

Comment:
yeah sidekiq is like a good unix program: https://github.com/mperham/sidekiq/wiki/Signals

I think there are other ways, i saw a sidekiqctl thing, but this seems more straightforward /shrug.

ideally we would send a USR1 when we start our deploy to tell it to stop accepting work, but it doesn't seem necessary for the moment